libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.html

```diff
@@ -3,7 +3,7 @@
   aria-haspopup="listbox"
   class="sky-autocomplete"
   role="combobox"
-  [attr.aria-expanded]="isOpen && searchText"
+  [attr.aria-expanded]="isOpen && !!searchText"
```

Comment:
Talked offline - lets remove the `&& !!searchText` here.

Reply:
done in: [remove unnecessary qualifier for aria-expanded](https://github.com/blackbaud/skyux/pull/544/commits/bafa3e574155dc6d3f8108e050308d86dcd289ad)